src/epcpm/mainwindow.ui

```diff
@@ -121,7 +121,24 @@
             <widget class="QTreeView" name="sunspec2_view"/>
            </item>
            <item row="0" column="0">
-            <widget class="SearchBox" name="sunspec2_search_box" native="true"/>
+            <widget class="SearchBox" name="sunspec2_search_box"/>
+           </item>
+          </layout>
+         </item>
+        </layout>
+       </widget>
+       <widget class="QWidget" name="tab_8">
+        <attribute name="title">
+         <string>Anomalies</string>
+        </attribute>
+        <layout class="QHBoxLayout" name="horizontalLayout_3">
+         <item>
+          <layout class="QGridLayout" name="gridLayout_3">
+           <item row="1" column="0">
+            <widget class="QTreeView" name="anomalies_view"/>
+           </item>
+           <item row="0" column="0">
+            <widget class="SearchBox" name="anomalies_search_box_2"/>
```

Comment:
The anomalies search box does not work. I'm unsure which column it should be searching on?
```suggestion
            <widget class="SearchBox" name="anomalies_search_box"/>
```

Reply:
Fixed, will search on name column